Editor/Modules/AssetsModule.cs

```diff
@@ -109,7 +109,7 @@ static void AnalyzeResources(IList<ProjectIssue> issues)
             }
         }
 
-        static void AnalyzeStreamingAssets(IList<ProjectIssue> issues)
+        static void AnalyzeStreamingAssets(ProjectAuditorSettings settings, IList<ProjectIssue> issues)
```

Comment:
Here one way we could provide settings. The module code already used the provider and get the platform-specific settings.

In general any call to Analyze() for each analyzer could also contain either the settings or the provider, as I did so far (since the Analyze() calls already contain the platform as a parameter, thus they can get the settings).